### PR TITLE
Update meteoalarm.markdown

### PR DESCRIPTION
**Pull request by rolfberkenbosch**
## Proposed change

MeteoAlarm has changed their api and website, so i made a small change in the documentation and intergration to make it work again.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [x] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: 

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/meteoalarm.markdown [hunk 1]
```diff
@@ -34,7 +34,7 @@ name:
   default: meteoalarm
   type: string
 country:
-  description: The 2 letters of your country
+  description: The full name of your country in english format
```
Comment (hmmbob) [inline source/_integrations/meteoalarm.markdown]:
```suggestion
  description: The full name of your country in English format
```

File: source/_integrations/meteoalarm.markdown [hunk 2]
```diff
@@ -23,7 +23,7 @@ To enable this binary sensor, add the following lines to your `configuration.yam
 ```yaml
 binary_sensor:
   - platform: meteoalarm
-    country: "NL"
+    country: "netherlands"
```
Comment (hmmbob) [inline source/_integrations/meteoalarm.markdown]: Sure it shouldn't be `Netherlands`? I think to see the website using capitalized country names

Comment (rolfberkenbosch) [reply source/_integrations/meteoalarm.markdown]: It should be in lowercases, see: https://feeds.meteoalarm.org/feeds/meteoalarm-legacy-atom-netherlands

File: source/_integrations/meteoalarm.markdown (hunk 1 above)
Comment (rolfberkenbosch) [reply source/_integrations/meteoalarm.markdown]: It should be in lowercases, see: https://feeds.meteoalarm.org/feeds/meteoalarm-legacy-atom-netherlands

File: source/_integrations/meteoalarm.markdown (hunk 2 above)
Comment (hmmbob) [reply source/_integrations/meteoalarm.markdown]: Ok. Maybe nice to catch this in the upstream package? so if someone capitalizes the country, you just make lower_case out of it anyway?

Comment (hmmbob) [reply source/_integrations/meteoalarm.markdown]: Never mind, it's already there:

https://github.com/rolfberkenbosch/meteoalert-api/blob/master/meteoalertapi/meteoalertapi.py#L14

**Comment by rolfberkenbosch** [comment]
And my commit it is gone again.......

**Comment by hmmbob** [comment]
Maybe get rid of the pull bot?

**Comment by rolfberkenbosch** [comment]
I'm closing this one now because it has no commits. I created a new one from an new branch, I think my commit messages will be overwritten by the home-assistant branch when someone else is doing also an commit, and that commit is merged.

**Comment by hmmbob** [comment]
Remove the commit bot from your own repo.

**Comment by rolfberkenbosch** [comment]
> Remove the commit bot from your own repo.

You are absolutely right, it was the commit bot. I forgot to make a new branch and add from that new branch a commit 🤦. I have create a new commit from a seperate branch, sorry for my stupid mistakes. This is my new PR: https://github.com/home-assistant/home-assistant.io/pull/18075

**Pull request closed without merging**